manifests/0000_50_cluster-monitoring-operator_04-config.yaml

```diff
@@ -800,6 +800,26 @@ data:
     #
     # Status of the noobaa service in the addon's namespace.
     - '{__name__="namespace:noobaa_system_health_status:max"}'
+    #
+    # owners: (@mt-sre, Managed Tenants SRE)
+    #
+    # Total number of Addon installations, grouped by 'available', 'paused' and 'total'
+    - '{__name__="addon_operator_addons_count"}'
+    #
+    # owners: (@mt-sre, Managed Tenants SRE)
+    #
+    # A boolean that tells if the AddonOperator is paused
+    - '{__name__="addon_operator_paused"}'
+    #
+    # owners: (@mt-sre, Managed Tenants SRE)
+    #
+    # OCM API request latencies in microseconds
+    - '{__name__="addon_operator_ocm_api_requests_durations"}'
```

Comment:
Just an observation, this is a histogram.  Quick look at a stage cluster had `quantile={0.5,0.9,0.99}` so total metrics for the PR is 6